### PR TITLE
docs: Add Phrase integration section to `multi-language` docs, fix docs styling issues

### PR DESCRIPTION
**Pull request by askoufis**
We never actually had documentation about sku's `translations push/pull` commands, I think they just spread by word of mouth, or people figured them out from the `vocab` equivalents. I've also "documented" the tags feature by linking to the appropriate section of the vocab docs.

This PR also fixes the styling issues with the docs site.
See commit messages for those changes.

### CODE REVIEW

File: docs/docs/multi-language.md [hunk 1]
```diff
@@ -127,6 +129,52 @@ When formatting ICU messages vocab will format values such dates and numbers acc
...
+$ sku translations push --delete-unused-keys
+```
+
+### Tags
+
+The Phrase platform allows you to attach tags to translation keys.
+
+sku will push any tags present in your `translations.json` file to phrase when the `sku translations push` command is used.
+
+See the [Vocab documentation](https://github.com/seek-oss/vocab#Tags) for how to
```
Comment (jahredhope) [inline docs/docs/multi-language.md:176]: Unfinished or missing full stop?

Comment (askoufis) [reply docs/docs/multi-language.md:176]: Unfinished 😅

File: docs/docs/multi-language.md [hunk 2]
```diff
@@ -127,6 +129,52 @@ When formatting ICU messages vocab will format values such dates and numbers acc
...
+## Translation Platforms
+
+sku supports specific features from 3rd-party translation platforms.
+Currently the only translations platform supported by sku is [Phrase](https://phrase.com/).
+
+## Phrase-specific Features
+
+### Translation Syncing
+
+sku can be used to synchronize your translations to and from .
```
Comment (michaeltaranto) [inline docs/docs/multi-language.md:141]: This sentence looks incomplete

Comment (askoufis) [reply docs/docs/multi-language.md]: 🤦 Fixed

**Review by michaeltaranto — APPROVED** [review]

**Pull request merged**